### DISCUSSION TITLE
useSWRConfig can't be imported

### DISCUSSION THREAD

**Discussion by sanjayrjs16** [vercel/swr#1501] [Q&A]
I am trying to import `useSWRConfig` to make uses of `mutate` function.
`import useSWR, { useSWRConfig } from 'swr';`
But I get the following error

> Module '"swr"' has no exported member 'useSWRConfig'. Did you mean to use 'import useSWRConfig from "swr"' instead?

Still not working when I tried ` import useSWRConfig from "swr"`.
Am I missing something here ?

**Comment by shuding** [comment]
Which version are you using? `useSWRConfig` is only available for >=1.0 versions.

**Reply by sanjayrjs16** [reply]
Right, Got it.
I'm using `0.5.6`. 
Figured out the solution we can import `mutate` directly
`import useSWR, { mutate } from "swr"`
Thanks @shuding

**Reply by shuding** [reply]
It might be worth upgrading! Let me know if you have any issue with that :)
